plugins/auto-sizes/includes/improve-calculate-sizes.php

```diff
@@ -99,7 +99,7 @@ function auto_sizes_filter_image_tag( $content, array $parsed_block, WP_Block $b
 			$alignment                = $block->attributes['align'] ?? '';
 			$width                    = isset( $block->attributes['width'] ) ? (int) $block->attributes['width'] : 0;
 			$max_alignment            = $block->context['max_alignment'] ?? '';
-			$container_relative_width = $block->context['container_relative_width'] ?? '';
+			$container_relative_width = $block->context['container_relative_width'] ?? '1.0';
```

Comment:
Is `$block->context['container_relative_width']` a float if not null? Instead of a string `'1.0'` shouldn't a float be used here?

```suggestion
			$container_relative_width = $block->context['container_relative_width'] ?? 1.0;
```

Reply:
Yes it's float value. In https://github.com/WordPress/performance/pull/1818/commits/90dc20fa67fad5df900368bd31ebec6f0ac09b2b i update it.